config/webpack.config.js

```diff
@@ -106,6 +106,14 @@ const configGenerator = (options) => {
         {
           test: /\.json$/,
           loader: 'json-loader'
+        },
+        {
+          test: /react-jsonschema-form\/lib\/components\/(widgets|fields\/ObjectField|fields\/ArrayField)/,
```

Comment:
Yeah, it's a little weird. The loader here sets files to null. So the `test` regex is telling you what files you want the loader to set to null (i.e. exclude). The `exclude` array specifies what files to exclude from being processed by the loader (i.e. set to null and excluded). So the `exclude` array is really specifying what files we still want to keep, even though they match the `test` regex.